packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -222,6 +222,9 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
       }
       case 'Home':
         if (delegate.getFirstKey) {
+          if (manager.focusedKey === null && e.shiftKey) {
+            return;
+          }
```

Comment:
I think this is fine, but I wonder if we should make it consistent with PageUp/Down where Home/End returns early if the `focusedKey` is null (or if PageUp/Down should be changed so they move focus into the collection even if a focusedKey doesn't exist). Also, I remember that you mentioned noticing an inconsistency between RAC and RSP combobox or something? Mind reminding me what that was?

Reply:
I'm not noticing the inconsistency between RAC and RSP for this behavior using the current builds for this PR now, but perhaps it's because I've gotten used to a particular way of testing. 

Regarding PageUp/PageDown, I'm not sure that adding this code will change anything in the behavior when there is no focusedKey, because, with no focusedKey, PageUp/PageDown currently doesn't seem to do anything.

It may make sense for PageUp/PageDown to move focus into the collection as you suggest.

Reply:
I also spent some time yesterday seeing if I could change the behavior of the Escape key so that it does not clear the input if the user wants to close the input before editing their search.